### ISSUE TITLE
Please add a search button

### ISSUE THREAD

**Issue by iBabu007** [emavgl/oinkoin#160]
I propose adding a search button to the Expense App to enhance user experience by allowing users to quickly find specific expenses from their transaction history.
Implementing this feature will streamline the expense tracking process, making it more efficient and user-friendly.

**Comment by riveravaldez** [comment]
Absolutely. To be able to filter and search is really a must in order to make the app more useful. 💟

**Comment by DuckDuckWhale** [comment]
It's quite sad that the best Linux best transaction tracking app that I found (Denaro) can search but can't be used on mobile, adding a barrier to tracking every transaction, while the Android transaction tracking app is convenient to input a more complete transaction history yet can't search for transactions, and by doing so makes it hard to gain insights from this more complete history, which is the very point of doing tracking to begin with.

**Comment by riveravaldez** [comment]
Awesome! Thanks a lot,for the implementation and for let everybody know! <3
Hope you have a great day. ^_^
